### DISCUSSION TITLE
use Wireguard for local network access together with OpenClash

### DISCUSSION THREAD

**Discussion by hadifarnoud** [vernesong/OpenClash#3650] [Q&A] [answered]
I use OpenClash to bypass internet censorship. I also need a Wireguard to access my home network remotely. I installed WG via [this script](https://github.com/onemarcfifty/openwrt-scripts/blob/master/wireguard_install.sh) but lost all internet access.

How can I configure Wireguard in a way that does not interfere?

**Comment by hadifarnoud** [comment]
anyone?

**Comment by vernesong** [comment, marked as answer]
![image](https://github.com/vernesong/OpenClash/assets/42875168/c6012eee-c5a7-4642-a677-000d16114fa5)
Try this option by fill 51820, if this not work, show me your complete firewall rules what include oc and wg

**Reply by hadifarnoud** [reply]
thanks a lot
